pkg/virt-config/config-map.go

```diff
@@ -252,6 +260,16 @@ func (c *ClusterConfig) SetConfigModifiedCallback(cb ConfigModifiedFn) {
 	go c.configModifiedCallback()
 }
 
+// SmbiosConfig struct to take the mentioned values from Kubevirt-Config
+// and same values are synced with VMs
+type SmbiosConfig struct {
```

Comment:
I wondering if we can/should use the generated struct here as well? It also has the json tags, so it shoiuld work oyt of the box.

Reply:
I just confirmed that we can, so we should :)
Changed this.